src/main/resources/application.properties

```diff
@@ -4,7 +4,7 @@ spring.jackson.default-property-inclusion=non_null
 spring.application.name=ei-frontend
   #${project.artifactId}
 
-server.port=8080
+server.port=8086
```

Comment:
move it back to 8080

Reply:
Well, I will change it before merging.... right now I am testing(repeatedly running) the code in Travis to find out any failures and hence the causes.